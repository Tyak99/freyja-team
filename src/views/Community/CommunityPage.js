import React from 'react';
import Community from '../../components/Community/Community';

export default () => <Community />;
