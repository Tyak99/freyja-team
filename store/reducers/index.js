import { combineReducers } from 'redux';
import { createBrowserHistory } from 'history';
import profile from './profile';
import authReducer from './authReducer';
import modulesReducer from './modulesReducer';
import communityM from './community';

export const history = createBrowserHistory();

const appReducer = combineReducers({
  auth: authReducer,
  profile,
  modules: modulesReducer,
  community: communityM,
});

export default (state, action) => appReducer(state, action);
