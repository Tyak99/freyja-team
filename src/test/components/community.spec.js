import React from 'react';
import { shallow } from 'enzyme';
import Community from '../../components/Community/Community';
import store from '../../../store';

describe('test community component', () => {
  it('should render component', () => {
    const comm = shallow(<Community store={store} />);
    expect(comm).toMatchSnapshot();
  });
});
