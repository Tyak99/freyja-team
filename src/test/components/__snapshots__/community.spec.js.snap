// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test community component should render component 1`] = `ShallowWrapper {}`;
